cpp/README.md

```diff
@@ -3,7 +3,6 @@ cuStrings
 
 ### API Design
 
```

Comment:
I'm fine leaving this blank. I know it's not a design doc, but since it was previously pointing at an API doc would you prefer we update this to use https://docs.rapids.ai/api/libnvstrings/nightly/ instead of remove?

Reply:
Awesome! I did not realize this had been done yet. Yes, I will update the readme to point to this link.
Thanks Nick.